### PR TITLE
Rq3 brca refactoring newformat

### PR DESCRIPTION
**Pull request by shilpigoeldev**
Birmingham BRCA with new rules added and QAed by Fiona.

### CODE REVIEW

File: lib/import/helpers/brca/providers/rq3/rq3_helper.rb [hunk 1]
```diff
@@ -7,6 +7,22 @@ module Rq3
           module Rq3Helper
             include Import::Helpers::Brca::Providers::Rq3::Rq3Constants
 
+            def check_positive_record?
+              ['P', '?', 'UV', 'PATHOGENIC'].include? @posnegtest.upcase
```
Comment (ollietulloch) [inline lib/import/helpers/brca/providers/rq3/rq3_helper.rb:11]: @shilpigoeldev - could we use %w[] here?

File: test/lib/import/brca/providers/birmingham/birmingham_handler_newformat_test.rb [hunk 2]
```diff
@@ -58,6 +58,30 @@ def setup
...
+    assert_equal 8, genotypes[0].attribute_map['gene']
+    assert_equal 7, genotypes[1].attribute_map['gene']
+    assert_equal 2, genotypes[1].attribute_map['teststatus']
+    assert_equal 3, genotypes[1].attribute_map['variantpathclass']
+    assert_equal 'p.Thr1256Argfsx', genotypes[1].attribute_map['proteinimpact']
+  end
+
+  test 'negative_tests_from_fullscreen_newformat' do
+    negative_record_newformat = build_raw_record('pseudo_id1' => 'bob')
+    negative_record_newformat.raw_fields['overall2'] = 'Normal'
```
Comment (ollietulloch) [inline test/lib/import/brca/providers/birmingham/birmingham_handler_newformat_test.rb:75]: Could this be including in the `build_raw_record` call above? There are a few other examples.

**Review by ollietulloch — CHANGES_REQUESTED** [review]
Hi @shilpigoeldev - justa  couple of small changes. Thanks

File: lib/import/helpers/brca/providers/rq3/rq3_helper.rb (hunk 1 above)
Comment (shilpigoeldev) [reply lib/import/helpers/brca/providers/rq3/rq3_helper.rb]: Thanks @ollietulloch , done now .

File: test/lib/import/brca/providers/birmingham/birmingham_handler_newformat_test.rb (hunk 2 above)
Comment (shilpigoeldev) [reply test/lib/import/brca/providers/birmingham/birmingham_handler_newformat_test.rb:75]: `build_raw_record` creates a skeleton structure , for which we change the attributes as per the test case requirements and then do the assertions. Same is being used above too to build a negative record of new format.

**Review by ollietulloch — APPROVED** [review]

**Pull request merged**